tests/test_helpers/helpers.ts

```diff
@@ -173,6 +177,19 @@ export function makeTestEnv(
     askConfirmation: mockEnv.askConfirmation || (() => {}),
   });
 
+  // For tests without the grid composer mounted, we register fake composer
+  const composerFocusStore = container.get(ComposerFocusStore);
+  composerFocusStore.focusComposer(
```

Comment:
This wouldn't be necessary with the "void composer" idea, would it ?

Reply:
It still is with the VOID_COMPOSER that throws